aea/cli/core.py

```diff
@@ -77,11 +79,15 @@
     default=False,
     help="Skip consistency checks of agent during command execution.",
 )
+@registry_path_option
 @click.pass_context
-def cli(click_context, skip_consistency_check: bool) -> None:
+def cli(
+    click_context, skip_consistency_check: bool, registry_path: Optional[str]
+) -> None:
     """Command-line tool for setting up an Autonomous Economic Agent (AEA)."""
     verbosity_option = click_context.meta.pop("verbosity")
     click_context.obj = Context(cwd=".", verbosity=verbosity_option)
+    click_context.obj.registry_path = registry_path
```

Comment:
By default, registry path is defined via aea-config. So now we're potentially overwriting this with `None`

Reply:
for example in `aea add` it used as this. so it can be overridden by the option if needed, By default, it's None and should not cause a problem, cause ctx.agent_config.registry_path is used directly.


```
# check in registry
    registry_path = (
        os.path.join(ctx.cwd, ctx.agent_config.registry_path)
        if ctx.registry_path is None
        else ctx.registry_path
    )
```